extensions/wikia/UserLogin/FacebookSignupController.class.php

```diff
@@ -279,27 +301,25 @@ protected function getValidFbUserId() {
 	 * @return null|User
 	 */
 	protected function getValidWikiaUser( $wikiaUserName, $wikiaPassword ) {
```

Comment:
This nested conditional was more confusing than helpful.  Split it out into individual checks.